managed/services/dbaas/kubernetes/kubernetes.go

```diff
@@ -465,9 +465,6 @@ func (k *Kubernetes) GetAllClusterResources(ctx context.Context, clusterType Clu
 		switch clusterType {
 		case ClusterTypeUnknown:
 			return 0, 0, 0, errors.Errorf("unknown cluster type")
-		case ClusterTypeGeneric:
-			// TODO support other cluster types
-			return 0, 0, 0, nil
```

Comment:
I suggest we keep this case but remove the action and leave a comment stating that we require it to be a no-op to execute the logic after the switch case.

Reply:
Done. It just continue looping